bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/validation/zeebe/SubProcessValidator.java

```diff
@@ -86,7 +88,7 @@ private void validateEventSubprocess(
           if (SUPPORTED_START_TYPES.stream().noneMatch(type -> type.isInstance(def))) {
             validationResultCollector.addError(
                 0,
-                "Start events in event subprocesses must be one of: message, timer, error, signal");
+                "Start events in event subprocesses must be one of: message, timer, error, signal, escalation");
```

Comment:
❌ Please also add `escalation` to the error message above this. You could put the message in a variable and remove the duplication 🙂